packages/sources/css/src/components/button/src/index.css

```diff
@@ -164,19 +165,22 @@
 
   /* Sizes */
   .vtmn-btn_size--small {
-    font-size: 0.75rem;
-    padding: 0.5rem 1rem;
+    font-size: 0.875rem;
+    padding: 0.4375rem 1rem;
+    line-height: calc(1.125 / 0.875);
```

Comment:
Why you've setted a `calc` for the small class and not for the others (medium & large)?